packages/shared-ui/src/components/deploy-panel/DeploymentView.tsx

```
@@ -0,0 +1,573 @@
+		border: `1px solid ${on ? 'color-mix(in srgb, var(--rr-color-success) 40%, transparent)' : 'var(--rr-border)'}`,
+		background: on ? 'color-mix(in srgb, var(--rr-color-success) 10%, transparent)' : 'transparent',
+		color: on ? 'var(--rr-color-success)' : 'var(--rr-text-secondary)',
+	}),
+	configRow: {
+		display: 'flex',
+		alignItems: 'center',
+		gap: 12,
+		padding: '10px 14px',
+		borderBottom: '1px solid var(--rr-border)',
+		fontSize: 12.5,
+	} as CSSProperties,
```

Biome (2.5.5)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)